app/server/static/components/label.vue

```diff
@@ -176,13 +177,33 @@
                     a.button.is-primary(v-on:click="doneEdit(label)") Save changes
 </template>
 
+<style scoped>
+.random-color-button {
+  height: 36px;
+  width: 36px;
+  background-color: transparent;
+  color: #404040;
+  border: none;
+}
+
+.random-color-button-edit {
```

Comment:
This has the same CSS rules as the `.random-color-button` selector. Can the two selectors be merged? Either reference the same `.random-color-button` class in the template for both elements, or merge the CSS definitions:

```css
.random-color-button, .random-color-button-edit {
 ...
}
```

Reply:
Thanks for spotting this - now both uses reference the same `.random-color-button` class.